admin_guide/quota.adoc

```diff
@@ -149,6 +149,10 @@ A *BestEffort* scope restricts a quota to limit the following resources:
 
 - `*pods*`
 
+*BestEffort* scoped pods can take advantage of slack compute resources in their
+namespace, and then require that users make an explicit resource request for
+pods that require a higher quality of service.
```

Comment:
this still doesn't really tell me how i create a besteffort pod.  

The other page says that the BestEffort scope will "Match pods that have best effort quality of service for either cpu or memory."

I still don't understand how I set memory or cpu to "best effort"  (the reality, I believe, is that if you set a limit, but no request, for a given resource(eg memory or cpu) that is considered best effort for that resource, but I don't think we state that explicitly anywhere)